entities/config/helpers.go

```diff
@@ -21,3 +25,19 @@ func Enabled(value string) bool {
 		return false
 	}
 }
+
+var defaultMinimumTimeout = -1 * time.Second
+
+func MinimumTimeout() time.Duration {
```

Comment:
Can we make this setting a part of our `config.Config` struct? like we have there `ModuleHttpClientTimeout` for modules, we could have there `ClusterApiClientTimeout` that would indicate it's purpose more clearly

Reply:
It's not just the clusterApi, it's also the search timeout.

The CSEs want to be able to run longer queries, even if they use a lot of resources.